extensions/azurePublishNew/src/node/backgroundProcessManager.ts

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
I added this just for now to satisfy the provision method in the plug-in.  We need to refactor this to be more functional which I can do if/when I take on the telemetry and comm back to the client.